acmwebsite/public/js/site.js

```diff
@@ -1,22 +1,22 @@
 $(document).ready(function() {
-    function change_theme(theme_name) {
-        $('#bs-css').attr('href', '/css/bootstrap.' + theme_name + '.min.css');
-        $('#toggle-theme').text('Too ' + theme_name + '?');
-    }
+  function change_theme(theme_name) {
+    $('#bs-css').attr('href', '/css/bootstrap.' + theme_name + '.min.css');
+    $('#toggle-theme').text('Too ' + theme_name + '?');
+  }
 
-    $('#toggle-theme').click(function (event) {
-        event.preventDefault();
-        $.get("/toggle_theme", change_theme, "text");
-    });
+  $('#toggle-theme').click(function (event) {
+    event.preventDefault();
+    $.get("/toggle_theme", change_theme, "text");
+  });
 
-    $(".vupdate").on("change keyup paste", function() {
-        $(this).attr("value", $(this).val());
-    });
+  $(".vupdate").on("change keyup paste", function() {
+    $(this).attr("value", $(this).val());
+  });
 
-    $('input[name="first_time"]').change(function() {
-        checked = this.checked;
-        $('.on-first-time').each(function() {
-            if (checked) { $(this).fadeIn() } else { $(this).fadeOut() }
-        });
+  $('input[name="first_time"]').change(function() {
+    checked = this.checked;
+    $('.on-first-time').each(function() {
+      if (checked) { $(this).fadeIn() } else { $(this).fadeOut() }
```

Comment:
Same here... not sure how this is related to your PR.

Reply:
Let's stick to 4 spaces for indentation on all code, two spaces on markup. Seems more consistent than this silly 2-space JavaScript thing.

Reply:
I've updated the code. However, you can't just go around changing the contributing guidelines on a whim like this again. It is no longer just you, Sam, and me on the project.

In the future, please inform all contributors (by email or some other method) before changing the contributing guidelines so that we can all be on the same page.